articles/20240828_Daytona_Office_Hours_#8:_Latest_Updates_and_Highlights.md

```diff
@@ -0,0 +1,59 @@
+- **Server Configuration Directory Fix:** Corrects directory creation for remote profiles. Previously, when you modified a directory path in the server configuration for a remote profile, Daytona would automatically attempt to create the directory on your local machine. This update now prevents unnecessary directory creation on the host machine when using remote profiles.
+- **Daytona Creation Logs:** We've fixed an issue that affected Daytona creation logs. Now, when you create a new Daytona project, the logs will display correctly, ensuring a smoother development experience while working on Daytona inside Daytona.
+- **Proper API JSON validation:** We've implemented a significant change to our API's Swagger specification. This update ensures strict JSON validation, requiring all API requests to adhere to a precise structure.
+  
+## Breaking Changes
+
+*We've significantly improved our API's Swagger specification to include strict validation for required properties. This change ensures that API clients generated from our Swagger definition adhere to the correct structure, preventing potential errors and inconsistencies. While this is the only breaking change in this release, it's essential for maintaining API consistency and reliability.*
+
+## Patches
+
+- **Different Config Directory:** To avoid conflicts when working on Daytona within a Daytona project, we now use a different configuration directory in development mode. This allows you to have distinct settings for development and production environments.
+- **Purge Command Enhancement:** We've simplified the purge command. Now, instead of manually stopping and starting the Daytona server, the command automatically handles these steps and efficiently deletes all workspaces. Additionally, we've fixed an issue that prevented the local container registry volume from being wiped clean.
+
+## Announcing Pre-Builds and Multiple IDs Support For Git Providers and More!
+
+- **Pre-Builds Feature:** Ability to configure pre-builds for quicker startup times. Accelerates project startup with pre-built images or dev containers. This feature eliminates the need for lengthy build processes, allowing you to dive into your projects more quickly.
+- **Multiple Identities Support for Git Providers:** Support users with multiple GitHub accounts. This will provide greater flexibility and convenience for users working with multiple accounts.
+- **Commit URL Clone Fix:** Ensures successful cloning of repositories from non-main branches.
+
+## Conclusion
+
+To conclude this episode of Daytona Office Hours, we explored the latest updates and exciting developments, from essential quality-of-life improvements to new features like *server port notifications* and the upcoming *builds runner*. As we look ahead, we're excited about the potential of pre-builds and multi-GitHub account support.
```

Comment:
This is not natural to read from the article. And it is essentially rehashing above. Also it is pure AI slop. Reread the whole thing and edit for better readability, please do it manually, remove what needs to be removed.

Reply:
Apologies, used AI to rephrase the conclusion but this time it's completely manual. Thank you!